QUEST_LV_0200.tsv

```diff
@@ -3908,7 +3908,7 @@ QUEST_LV_0200_20150323_003907	I would say that was the wisdom of Maven.{nl}Maven
 QUEST_LV_0200_20150323_003908	So if we could just recharge the magical power, any kinds of magics will be okay.{nl}Even if that's a magical power from the demons.
 QUEST_LV_0200_20150323_003909	Go to Veiniki altar when you recharge the sacred symbol of the spiritual power.{nl}The third key is waiting for you.
 QUEST_LV_0200_20150323_003910	{memo X}If you think you are ready, I will show you the true nature of the curse.{nl}If you can't eliminate it at once, it will make a bigger trouble.
-QUEST_LV_0200_20150323_003911	So now, the last verification is left.{nl}The scripture that will possess my spirit, Please open it in front of the altar of the revelation.
+QUEST_LV_0200_20150323_003911	$Now all that's left is the Verification Test.{nl}The scripture that possesses my spirit, please open it in front of the altar of the revelation.
```

Comment:
Aurelius and the Cathedral treat it like a formality. You basically duel with a 'champion of the gate' type figure and after beating him to 10% hp the gate opens.